src/main/java/build/buildfarm/instance/stub/StubInstance.java

```diff
@@ -132,6 +134,7 @@
   private final TimeUnit deadlineAfterUnits;
   private final Retrier retrier;
   private final @Nullable ListeningScheduledExecutorService retryService;
+  private final ExecutorService byteStreamExecutorService;
```

Comment:
Provide the member level initialization here of `newSingleThreadExecutor()`, rather than in the constructor, as it's not dependent upon parameter injection

Reply:
Done